src/main/java/org/jabref/gui/edit/CopyMoreAction.java

```diff
@@ -224,7 +224,7 @@ private void copyKeyAndLink() {
             keyAndLink.append(OS.NEWLINE);
         }
 
-        clipBoardManager.setHtmlContent(keyAndLink.toString());
+        clipBoardManager.setContent(keyAndLink.toString());
```

Comment:
This really puts the html string into the clipboard, so if you paste it into say notepad you get `<a href="url">key</a>`. Instead, its better to add the html, and in addition also a fallback string of the format `key - url`. In this way, you get the desired behavior also when you paste in applications that do understand html (e.g onenote).

In order for this to work, you probably need to add a second fallback argument to
https://github.com/JabRef/jabref/blob/3db431321186535cc9a05ceae8b1fa580585a9c0/src/main/java/org/jabref/gui/ClipBoardManager.java#L140-L145
which is then added to the ClipboardContent via `putString`.

Reply:
Hi @tobiasdiez !
Ah ok that makes sense.. I guess I should have looked deeper into what setHtmlContent() actually does.. I assumed it would be applicable for url's on their own only. So that means, both, setContent() and setHtmlContent() should be called? 
Sorry for my noobness! This was my first open-source contribution ever :D
Can I fix the issue tonight (German time) and commit the changes?
thanks,
Niko

Reply:
No worries, you were on the right track. Calling `setContent` and `setHtmlContent` is the right idea, but I suspect that this will push two different clipboard items. Instead, the `setHtmlContent` should be modified as follows:
```
public void setHtmlContent(String html, String fallbackPlain) { 
     final ClipboardContent content = new ClipboardContent(); 
     content.putHtml(html);
     content.putString(fallbackPlain);
     clipboard.setContent(content); 
     setPrimaryClipboardContent(content); 
 }
```
This should add one clipboard item, containing both the html as well as the normal string. (I hope)

Let me know if you encounter any further questions. There is not such a thing as noobness. We all can learn more. And learning in a relaxed environment is one of the main points of open source, so don't worry. Happy coding!

Reply:
> but I suspect that this will push two different clipboard items.
That was exactly my next thought :)
Ok, I will try..
Thanks for the kind words!

Reply:
alright, i hope i did what you meant. I have to say, I still don't understand why copyKeyAndLink() creates html content for the clipboard, but e.g. copyKeyAndTitle() does not?
cheers,
Niko